schemas/users.js

```diff
@@ -1,18 +1,20 @@
 const Joi = require("joi")
 const emailRegexp = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+const subscriptionList = ["starter", "pro", "business"];
 
 const registerSchema = Joi.object({
   name: Joi.string().required(),
   email: Joi.string().pattern(emailRegexp).required(),
   password: Joi.string().min(6).required(),
-});
+  subscription: Joi.string().valid(...subscriptionList).required(),
+})
 
 const loginSchema = Joi.object({
-    email: Joi.string().pattarn(emailRegexp).required(),
-    password: Joi.string().min(6).required(),
-});
+  email: Joi.string().pattern(emailRegexp).required(),
```

Comment:
```suggestion
  email: Joi.string().email().required(),
```
Для перевірки email в Joi передбачений спеціальний метод. А от для паролю доцільно додати патерн